src/components/ScholarshipList.jsx

```diff
@@ -52,10 +134,12 @@ function ScholarshipList({ scholarships }) {
 }
 
 ScholarshipList.propTypes = {
-  scholarships: PropTypes.arrayOf(PropTypes.object),
+  listFn: PropTypes.func,
+  noResultsNode: PropTypes.node,
 };
 ScholarshipList.defaultProps = {
-  scholarships: [],
+  listFn: undefined,
+  noResultsNode: <Typography>No scholarships found</Typography>,
```

Comment:
ooo I didn't know we could set jsx as default props

Reply:
Yeah! The only thing tricky about it is when you need to set a `className`. In that case, I would set this `undefined` and when rendering set the value there.